src/main/java/racinggame/domain/exception/InputOverFiveLengthException.java

```diff
@@ -0,0 +1,14 @@
+package racinggame.domain.exception;
+
+public class InputOverFiveLengthException extends RuntimeException {
+
+    public static final String DEFAULT_MESSAGE = "이름은 5자를 초과 할 수 없습니다.";
```

Comment:
```suggestion
    public static final String DEFAULT_MESSAGE = "이름은 5자를 초과 할 수 없습니다.";
```
`5자`의 조건이 변경된다면 오류 발생시 디버깅하는데에 혼선이 있을것 같아요!
`5자`의 조건이 변경되면 여기에 와서 또한번 변경해야되는 이슈가 있을것 같아요! 🤔